src/H5Shyper.c

```diff
@@ -4821,7 +4821,7 @@ H5Sget_select_hyper_blocklist(hid_t spaceid, hsize_t startblock, hsize_t numbloc
     herr_t ret_value; /* return value */
 
     FUNC_ENTER_API(FAIL)
-    H5TRACE4("e", "ihhx", spaceid, startblock, numblocks, buf);
+    H5TRACE4("e", "ihh*[a2]h", spaceid, startblock, numblocks, buf);
```

Comment:
Interesting syntax here. Is this intended?

Reply:
Ha. No idea. I didn't add anything like that, so I'd have to check to see what's going on in the trace code in H5trace.c.